make failed with gtest

Did a git pull today and tried a `--enable-gtest` on a standalone machine (arm if that's important).  `make gtest` had been running happily thus far.

git master @ 95031af7e4943411b7d23327db0b24bc9c11908c

```shell
$ make
...
...
...
Making all in ucm/test_dlopen
make[3]: Entering directory '/home/tc/ucx-gtest/source/test/gtest/ucm/test_dlopen'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/tc/ucx-gtest/source/test/gtest/ucm/test_dlopen'
make[3]: Entering directory '/home/tc/ucx-gtest/source/test/gtest'
  CXXLD    gtest
/usr/bin/ld: uct/ib/gtest-test_ib.o: undefined reference to symbol 'ibv_free_device_list@@IBVERBS_1.1'
//usr/lib/aarch64-linux-gnu/libibverbs.so.1: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
Makefile:1044: recipe for target 'gtest' failed
make[3]: *** [gtest] Error 1
make[3]: Leaving directory '/home/tc/ucx-gtest/source/test/gtest'
Makefile:2567: recipe for target 'all-recursive' failed
make[2]: *** [all-recursive] Error 1
make[2]: Leaving directory '/home/tc/ucx-gtest/source/test/gtest'
Makefile:718: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/home/tc/ucx-gtest/source'
Makefile:586: recipe for target 'all' failed
make: *** [all] Error 2
```

Same,  I'm afraid.  Brand new git-clone.

```shell
$ make
...
...
make[3]: Entering directory '/home/tc/ucx-gtest/source/test/gtest'
/bin/bash ../../libtool  --tag=CXX   --mode=link g++ -O3 -g -Wall -Werror  -fno-tree-vectorize -DGTEST_UCM_HOOK_LIB_DIR="\"/home/tc/ucx-gtest/source/test/gtest/ucm/test_dlopen/.libs\""   -no-install -Wl,-dynamic-list-data  -o gtest common/gtest-gtest-all.o common/gtest-main.o common/gtest-test_helpers.o common/gtest-test_obj_size.o common/gtest-test_perf.o common/gtest-test.o ucm/gtest-malloc_hook.o uct/gtest-test_amo.o uct/gtest-test_amo_add_xor.o uct/gtest-test_amo_and_or.o uct/gtest-test_amo_cswap.o uct/gtest-test_amo_fadd_fxor.o uct/gtest-test_amo_fand_for.o uct/gtest-test_amo_swap.o uct/gtest-test_event.o uct/gtest-test_fence.o uct/gtest-test_flush.o uct/gtest-test_many2one_am.o uct/gtest-test_md.o uct/gtest-test_mm.o uct/gtest-test_mem.o uct/gtest-test_p2p_am.o uct/gtest-test_p2p_err.o uct/gtest-test_p2p_mix.o uct/gtest-test_p2p_rma.o uct/gtest-test_pending.o uct/gtest-test_progress.o uct/gtest-test_uct_ep.o uct/gtest-test_uct_perf.o uct/gtest-test_zcopy_comp.o uct/gtest-uct_p2p_test.o uct/gtest-uct_test.o uct/gtest-test_stats.o ucs/gtest-test_stats_filter.o uct/gtest-test_peer_failure.o uct/gtest-test_tag.o ucp/gtest-test_ucp_stream.o ucp/gtest-test_ucp_peer_failure.o ucp/gtest-test_ucp_atomic.o ucp/gtest-test_ucp_dt.o ucp/gtest-test_ucp_memheap.o ucp/gtest-test_ucp_mmap.o ucp/gtest-test_ucp_mem_type.o ucp/gtest-test_ucp_perf.o ucp/gtest-test_ucp_rma.o ucp/gtest-test_ucp_rma_mt.o ucp/gtest-test_ucp_tag_cancel.o ucp/gtest-test_ucp_tag_match.o ucp/gtest-test_ucp_tag_offload.o ucp/gtest-test_ucp_tag_mt.o ucp/gtest-test_ucp_tag_perf.o ucp/gtest-test_ucp_tag_probe.o ucp/gtest-test_ucp_tag_xfer.o ucp/gtest-test_ucp_tag.o ucp/gtest-test_ucp_context.o ucp/gtest-test_ucp_wireup.o ucp/gtest-test_ucp_wakeup.o ucp/gtest-test_ucp_fence.o ucp/gtest-test_ucp_sockaddr.o ucp/gtest-ucp_test.o ucp/gtest-ucp_datatype.o ucs/gtest-test_algorithm.o ucs/gtest-test_arbiter.o ucs/gtest-test_async.o ucs/gtest-test_callbackq.o ucs/gtest-test_class.o ucs/gtest-test_config.o ucs/gtest-test_datatype.o ucs/gtest-test_debug.o ucs/gtest-test_memtrack.o ucs/gtest-test_math.o ucs/gtest-test_mpmc.o ucs/gtest-test_mpool.o ucs/gtest-test_pgtable.o ucs/gtest-test_profile.o ucs/gtest-test_rcache.o ucs/gtest-test_memtype_cache.o ucs/gtest-test_stats.o ucs/gtest-test_strided_alloc.o ucs/gtest-test_string.o ucs/gtest-test_sys.o ucs/gtest-test_time.o ucs/gtest-test_twheel.o ucs/gtest-test_frag_list.o ucs/gtest-test_type.o ucs/gtest-test_log.o uct/ib/gtest-test_ib.o uct/ib/gtest-test_ib_md.o uct/ib/gtest-test_cq_moderation.o uct/ib/gtest-test_ib_xfer.o uct/ib/gtest-ud_base.o uct/ib/gtest-test_ud.o uct/ib/gtest-test_ud_slow_timer.o uct/ib/gtest-test_ud_pending.o uct/ib/gtest-test_ud_ds.o uct/ib/gtest-test_rc.o    ../../src/ucs/libucs.la ../../src/uct/libuct.la ../../src/ucm/libucm.la ../../src/ucp/libucp.la ../../src/tools/perf/libucxperf.la -fopenmp  ../../src/uct/ib/libuct_ib.la -lpthread -lrt -lrt  -lbfd -ldl
libtool: link: g++ -O3 -g -Wall -Werror -fno-tree-vectorize -DGTEST_UCM_HOOK_LIB_DIR=\"/home/tc/ucx-gtest/source/test/gtest/ucm/test_dlopen/.libs\" -Wl,-dynamic-list-data -o gtest common/gtest-gtest-all.o common/gtest-main.o common/gtest-test_helpers.o common/gtest-test_obj_size.o common/gtest-test_perf.o common/gtest-test.o ucm/gtest-malloc_hook.o uct/gtest-test_amo.o uct/gtest-test_amo_add_xor.o uct/gtest-test_amo_and_or.o uct/gtest-test_amo_cswap.o uct/gtest-test_amo_fadd_fxor.o uct/gtest-test_amo_fand_for.o uct/gtest-test_amo_swap.o uct/gtest-test_event.o uct/gtest-test_fence.o uct/gtest-test_flush.o uct/gtest-test_many2one_am.o uct/gtest-test_md.o uct/gtest-test_mm.o uct/gtest-test_mem.o uct/gtest-test_p2p_am.o uct/gtest-test_p2p_err.o uct/gtest-test_p2p_mix.o uct/gtest-test_p2p_rma.o uct/gtest-test_pending.o uct/gtest-test_progress.o uct/gtest-test_uct_ep.o uct/gtest-test_uct_perf.o uct/gtest-test_zcopy_comp.o uct/gtest-uct_p2p_test.o uct/gtest-uct_test.o uct/gtest-test_stats.o ucs/gtest-test_stats_filter.o uct/gtest-test_peer_failure.o uct/gtest-test_tag.o ucp/gtest-test_ucp_stream.o ucp/gtest-test_ucp_peer_failure.o ucp/gtest-test_ucp_atomic.o ucp/gtest-test_ucp_dt.o ucp/gtest-test_ucp_memheap.o ucp/gtest-test_ucp_mmap.o ucp/gtest-test_ucp_mem_type.o ucp/gtest-test_ucp_perf.o ucp/gtest-test_ucp_rma.o ucp/gtest-test_ucp_rma_mt.o ucp/gtest-test_ucp_tag_cancel.o ucp/gtest-test_ucp_tag_match.o ucp/gtest-test_ucp_tag_offload.o ucp/gtest-test_ucp_tag_mt.o ucp/gtest-test_ucp_tag_perf.o ucp/gtest-test_ucp_tag_probe.o ucp/gtest-test_ucp_tag_xfer.o ucp/gtest-test_ucp_tag.o ucp/gtest-test_ucp_context.o ucp/gtest-test_ucp_wireup.o ucp/gtest-test_ucp_wakeup.o ucp/gtest-test_ucp_fence.o ucp/gtest-test_ucp_sockaddr.o ucp/gtest-ucp_test.o ucp/gtest-ucp_datatype.o ucs/gtest-test_algorithm.o ucs/gtest-test_arbiter.o ucs/gtest-test_async.o ucs/gtest-test_callbackq.o ucs/gtest-test_class.o ucs/gtest-test_config.o ucs/gtest-test_datatype.o ucs/gtest-test_debug.o ucs/gtest-test_memtrack.o ucs/gtest-test_math.o ucs/gtest-test_mpmc.o ucs/gtest-test_mpool.o ucs/gtest-test_pgtable.o ucs/gtest-test_profile.o ucs/gtest-test_rcache.o ucs/gtest-test_memtype_cache.o ucs/gtest-test_stats.o ucs/gtest-test_strided_alloc.o ucs/gtest-test_string.o ucs/gtest-test_sys.o ucs/gtest-test_time.o ucs/gtest-test_twheel.o ucs/gtest-test_frag_list.o ucs/gtest-test_type.o ucs/gtest-test_log.o uct/ib/gtest-test_ib.o uct/ib/gtest-test_ib_md.o uct/ib/gtest-test_cq_moderation.o uct/ib/gtest-test_ib_xfer.o uct/ib/gtest-ud_base.o uct/ib/gtest-test_ud.o uct/ib/gtest-test_ud_slow_timer.o uct/ib/gtest-test_ud_pending.o uct/ib/gtest-test_ud_ds.o uct/ib/gtest-test_rc.o -fopenmp  ../../src/ucs/.libs/libucs.so ../../src/uct/.libs/libuct.so ../../src/ucm/.libs/libucm.so ../../src/ucp/.libs/libucp.so ../../src/tools/perf/.libs/libucxperf.a /home/tc/ucx-gtest/source/src/ucp/.libs/libucp.so /home/tc/ucx-gtest/source/src/uct/.libs/libuct.so /home/tc/ucx-gtest/source/src/ucs/.libs/libucs.so -lnuma /home/tc/ucx-gtest/source/src/ucm/.libs/libucm.so /opt/gcc/8.2.0/lib/../lib64/libstdc++.so ../../src/uct/ib/.libs/libuct_ib.so -lpthread -lrt -lbfd -ldl -fopenmp -Wl,-rpath -Wl,/home/tc/ucx-gtest/source/src/ucs/.libs -Wl,-rpath -Wl,/home/tc/ucx-gtest/source/src/uct/.libs -Wl,-rpath -Wl,/home/tc/ucx-gtest/source/src/ucm/.libs -Wl,-rpath -Wl,/home/tc/ucx-gtest/source/src/ucp/.libs -Wl,-rpath -Wl,/opt/gcc/8.2.0/lib/../lib64 -Wl,-rpath -Wl,/home/tc/ucx-gtest/source/src/uct/ib/.libs -Wl,-rpath -Wl,/opt/gcc/8.2.0/lib/../lib64 -Wl,-rpath -Wl,/usr/lib/ucx
/usr/bin/ld: uct/ib/gtest-test_ib.o: undefined reference to symbol 'ibv_free_device_list@@IBVERBS_1.1'
//usr/lib/aarch64-linux-gnu/libibverbs.so.1: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
Makefile:1047: recipe for target 'gtest' failed
make[3]: *** [gtest] Error 1
make[3]: Leaving directory '/home/tc/ucx-gtest/source/test/gtest'
Makefile:2585: recipe for target 'all-recursive' failed
make[2]: *** [all-recursive] Error 1
make[2]: Leaving directory '/home/tc/ucx-gtest/source/test/gtest'
Makefile:718: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/home/tc/ucx-gtest/source'
Makefile:586: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/test/gtest/Makefile.am
+++ b/test/gtest/Makefile.am
@@ -161,7 +161,10 @@ gtest_SOURCES += \
 	uct/ib/test_cq_moderation.cc \
 	uct/ib/test_ib_xfer.cc
 gtest_CPPFLAGS += \
-    $(IBVERBS_CPPFLAGS)
+	$(IBVERBS_CPPFLAGS)
+gtest_LDADD += \
+	$(IBVERBS_LDFLAGS) \
+	$(top_builddir)/src/uct/ib/libuct_ib.la
 if HAVE_TL_UD
 gtest_SOURCES += \
 	uct/ib/ud_base.cc \
@@ -182,8 +185,6 @@ if HAVE_RDMACM
 gtest_SOURCES += \
 	uct/ib/test_sockaddr.cc
 endif
-gtest_LDADD += \
-	$(top_builddir)/src/uct/ib/libuct_ib.la
 endif # HAVE_IB
 
 if HAVE_CUDA

